Makefile

```diff
@@ -125,23 +128,28 @@ start-integration: web/src/build/static/app.js bin/goalert bin/psql-lite bin/wai
 jest: node_modules 
 	yarn workspace goalert-web run jest $(JEST_ARGS)
 
-test: node_modules jest
+test: node_modules jest ## Run all unit tests
 	go test -short ./...
 
 force-yarn:
 	yarn install --no-progress --silent --frozen-lockfile --check-files
 
-check: force-yarn generate node_modules
-	# go run ./devtools/ordermigrations -check
-	go vet ./...
-	go run github.com/gordonklaus/ineffassign ./...
-	CGO_ENABLED=0 go run honnef.co/go/tools/cmd/staticcheck ./...
+check: check-go check-js  ## Run all lint checks
+
+check-js: force-yarn generate node_modules
 	yarn run fmt
 	yarn run lint
 	yarn workspaces run check
 	./devtools/ci/tasks/scripts/codecheck.sh
 
-check-all: check test smoketest cy-wide-prod-run cy-mobile-prod-run
+check-go:
+	go vet ./...
+	go fmt ./...
+	# go run ./devtools/ordermigrations -check
+	go run github.com/gordonklaus/ineffassign ./...
+	CGO_ENABLED=0 go run honnef.co/go/tools/cmd/staticcheck ./...
+
+check-all: check test smoketest cy-wide-prod-run cy-mobile-prod-run ## Run all lint checks and integration tests
```

Comment:
Is it necessary to add a task that combines checks and tests? (if so we should come up with a better name) We already have all tests under a target `test-all` to put them into a single command:
```bash
make check test-all
```

Reply:
well, I actually didn't add this.  I only added a description as it seemed like something that you might want to publicly expose with make help.  I can remove this artifact if you want.

Reply:
removed